documentation/src/components/side-nav.ts

```diff
@@ -73,6 +75,7 @@ class SideNav extends LitElement {
                     </div>
                 </div>
                 <div id="navigation">
+                    <docs-search></docs-search>
```

Comment:
Please make this the last element in `id="nav-header"` so that the delivery more closely matches https://spectrum.adobe.com/ where the search bar is "fixed".

Reply:
Done